HomeWork2.java

```diff
@@ -0,0 +1,15 @@
+public class HomeWork2 {
+        public static void main(String[] args) {
+            byte a = 120;
+            byte b = -10;
+
+            int x = a + 50;
+
+
+            System.out.println(a + b);
+            System.out.println(x);
+            System.out.println(a > b);
+            System.out.println(a < b);
+
+        }
+}
```

Comment:
Task 1:

static void overloadedMethod(int input) {
System.out.println("The sum of each number for " + input + " is: " +
((input % 10) + ((input / 10) % 10) + ((input / 100) % 10)));
}

or another way of implementation

  public static int calculateDigitSum(int number) {
        int sum = 0;
        while (number > 0) {
            sum += number % 10;
            number /= 10;
        }
        return sum;
    }

Reply:
Task 2:

 public static int overloadedMethod(String word1, String word2) {
        int length1 = word1.length();
        int length2 = word2.length();
        return length1 + length2;
    }

Reply:
Task 3:

static boolean overloadedMethod(int number) {
if (number > 0) {
System.out.println("Number is greater than zero");
return true;
} else {
System.out.println("Number is not less than zero");
return false;
}
}

Reply:
Task 4:
we should call all methods (1, 2, 3 tasks) from the main method:

 public static void main(String[] args) {
        overloadedMethod( "Hi", "Bye");
    }